file.go

```diff
@@ -8,6 +8,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	WalkThroughError = "walking through file tree"
```

Comment:
I actually meant more like [this](https://github.com/mongodb/amboy/blob/3b1827bc716865525e0569d5e9186f166303a7bb/errors.go#L21-L50) which uses Go error types more idiomatically, but I guess this is fine, just for future reference.